cpp/cmake_modules/SetupCxxFlags.cmake

```diff
@@ -365,6 +365,7 @@ message("Configured for ${CMAKE_BUILD_TYPE} build (set with cmake -DCMAKE_BUILD_
 if ("${CMAKE_BUILD_TYPE}" STREQUAL "DEBUG")
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} ${C_FLAGS_DEBUG}")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CXX_FLAGS_DEBUG}")
+elseif ("${CMAKE_BUILD_TYPE}" STREQUAL "RELWITHDEBINFO")
 elseif ("${CMAKE_BUILD_TYPE}" STREQUAL "FASTDEBUG")
```

Comment:
Can you stack elseif in CMake like this?

Reply:
It has been working for me.
Python has `pass` which would disambiguate this; it's just an empty block.you'd like to suggest some flags which should be set in here?

Reply:
I see. It's fine like this, it just looked odd =)

Reply:
How about `-ggdb -O3 -DNDEBUG`?

BTW, I want to use `-g3` instead of `-ggdb`. Because `-ggdb` is the same as `-g` on most environments. And `-g3` provides macro expansion feature on GDB.

 https://gcc.gnu.org/onlinedocs/gcc-8.2.0/gcc/Debugging-Options.html#Debugging-Options